packages/types/src/context-types.ts

```diff
@@ -3,7 +3,7 @@ import { Plugin } from './plugin';
 import { Spread, TuplifyUnion, Unarray } from './utils';
 
 // We are using `interface` instead of `type` in order to allow type augmentation
-export interface DefaultContext extends Record<string | symbol | number, unknown> {}
+export interface DefaultContext {}
```

Comment:
Yeah, this's much cleaner. an empty interface already allows any keys and values by default